roles/uperf-bench/templates/server_vm.yml.j2

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  name: 'uperf-server-{{ item }}-{{ trunc_uuid }}'
+  namespace: '{{ operator_namespace }}'
+  labels:
+    app : uperf-bench-server-{{ item }}-{{ trunc_uuid }}
+    type : uperf-bench-server-{{ trunc_uuid }}
+spec:
+  domain:
+    cpu:
+      sockets: {{ uperf.server_vm.sockets }}
+      cores: {{ uperf.server_vm.cores }}
+      threads: {{ uperf.server_vm.threads }}
+      dedicatedCpuPlacement: {{ uperf.server_vm.dedicatedcpuplacement }}
+{% if 'hostpassthrough' in uperf.server_vm.extra_options %}
+      model: host-passthrough
+{% endif %}
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+        - name: default
+          {{ uperf.server_vm.network.front_end }}: {}
+      networkInterfaceMultiqueue: {{ uperf.server_vm.network.multiqueue.enabled }}
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: {{ uperf.server_vm.requests.memory }}
+      limits:
+        memory: {{ uperf.server_vm.limits.memory }}
+  terminationGracePeriodSeconds: 0
+{% if uperf.pin is sameas true %}
+  nodeSelector:
+    kubernetes.io/hostname: '{{ uperf.pin_server }}'
+{% endif %}
+  networks:
+    - name: default
+      pod: {}
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: {{ uperf.server_vm.image }}
+  - cloudInitNoCloud:
+      userData: |-
+        #cloud-config
+        password: ripsaw
+        chpasswd: { expire: False }
+        runcmd:
+{% if uperf.server_vm.network.multiqueue.enabled %}
+          - ethtool -L eth0 combined {{ uperf.server_vm.network.multiqueue.queues }}
+{% endif %}
+          - yum install -y yum-plugin-copr
+          - yum copr enable ndokos/pbench -y
+          - yum install -y pbench-uperf redis git
```

Comment:
Ugh. I wonder if there is a way we can do this from an outside script? maybe curl/wget a set of instructions on what to install? My concern is if something changes with pbench, we have to re-spin a new operator each time. 

As much as this doesn't sound like a problem, it became a problem with Browbeat. Every time someone would try the PBench work with Browbeat something was broken with the install of PBench... Just something we need to keep an eye on.

Reply:
We can look at using custom container-disk-images, but so far I've ran out of luck with it. maybe for subsequent iterations when there's enough adoption we could do that. But either way, given that it's iteration 0, i'd not try to worry about it rn.